### ISSUE TITLE
Remember full-screen mode across sessions

### ISSUE THREAD

**Issue by Eneen** [Beep6581/RawTherapee#3987]
There is similar option in Krita and in Blender. They remember if full-screen has been enabled and start in this mode after exit.

**Comment by dngimage** [comment]
Agree, RT should remember if full-screen has been enabled and also the main window's size and position.

The old versions (Win-64) used to remember windows's size and position. Unless it is a Gtk3 + Windows window manager  problem.

**Comment by Hombre57** [comment]
I voted against this proposal because it's a quite uncommon behavior. I never seen it before in fact.

**Comment by Eneen** [comment]
Krita, Blender and Natron use it (from apps I use)
